tests/reporter/kubernetes.go

```diff
@@ -76,6 +76,7 @@ type KubernetesReporter struct {
 
 type commands struct {
 	command        string
+	Args           []string
```

Comment:
I think the current structure should accommodate for the need here, without the need to extend it.
Most commands  have a multi word string, which should accommodate to what you are asking to do here.

Reply:
All commands represented by the `command` struct does not require running in a shell.
Once a command is more then executing a binary with some args is should run in a shell otherwise it wont interprate correctly and fail.

The way we reporter code use the command struct cannot work for command that requires a shell, thus it is extended to enable distinction between the shell binary and the args.